doc/rest_api/snippet_sharing/use_cases/UC_edit_user.txt

```diff
@@ -0,0 +1,17 @@
+Summary:
+----------------------------
+Title: edit user
+Scope: User Management
+Level: User Goal
+Actors: Authenticated user, Administrator
+Brief: Users can edit their own account information the same way Administrators can do it for them. Administrators can also change permissions of users.

```

Comment:
Haven't planned scoring and comments for the current version, but it is something that could be added quite easily later on I think.